app/models/park.rb

```diff
@@ -68,4 +69,12 @@ def self.create_nobori_10
       latitude: 34.393833,
     )
   end
+
+  def fee_check
+    if fee_changed?
```

Comment:
[rubocop] Use a guard clause instead of wrapping the code inside a conditional expression.